### DISCUSSION TITLE
List display & sorting

### DISCUSSION THREAD

**Discussion by Ziloxeur** [karitham/waifubot#56] [Ideas]
I just got some idea to improve the research of characters is someone list. It could be nice to implement a new argument in the command w.list (NOTA) so w.l NOTA. It would show a liste with only characters from the last argument (anime). If for some reason you couldn't do it as I said, just try to make w.list search (NOTA) that could be reduced in w.l s (NOTA). Addind the argument @mention would, as for each command, show the command for the person pinged. Don't hesitate to ask if you have any question or my explication wasn't clear.

PS : NOTA = Name of the anime

**Comment by karitham** [comment]
That would work, however it would require adding an additional field, (which isn't super hard considering I'm migrating to PostgreSQL, however, I'm not sure how to handle the case of the already existing characters.

I guess I can make a tool for querying already existing anime character's anime.

As for the ergonomic side, I think using reactions would be a better solution, where for example you add a specific reaction and the bot prompts you to enter an anime you want to query, and subsequently changes the displayed embed
